### ISSUE TITLE
Verbose mode reports incorrect entropy for an acrostic password

### ISSUE THREAD

**Issue by redacted** [redacted/XKCD-password-generator#85]
(External report)

if we choose "acrostic" mode - the word number is determined by the length of word we choose.
here's an example "Leon" - a 4 letter word

```
artur@elitebook ~ $ xkcdpass -V -a Leon
The supplied word list is located at /usr/lib/python3/dist-packages/xkcdpass/static/default.txt.
Your word list contains 38271 words, or 2^15.22 words.
A 6 word password from this list will have roughly 91 (15.22 * 6) bits of entropy,
assuming truly random word selection.
Lucknow evader onlooking noxiously
```

**Comment by AndresMWeber** [comment]
This seems to be fixed by #104 since the `options.numwords` initialization logic was moved to before the verbose printout.

```
D:\dev\XKCD-password-generator (andresmweber-bugfix/issue83-entropy-info) $ xkcdpass -V -a leon
With the current options, your word list contains 857 words.
A 4 word password from this list will have roughly 38 (9.74 * 4) bits of entropy,
assuming truly random word selection.

liqueur exalted operable natural
```

**Comment by redacted** [comment]
#104 has been merged in the latest release - thanks for flagging the issue!
